diff --git a/src/ffconf.h b/src/ffconf.h
index 045b3dc..c7b86e9 100644
--- a/src/ffconf.h
+++ b/src/ffconf.h
@@ -25,7 +25,7 @@
 /   3: f_lseek() function is removed in addition to 2. */
 
 
-#define FF_USE_STRFUNC  0
+#define FF_USE_STRFUNC  1
 /* This option switches string functions, f_gets(), f_putc(), f_puts() and f_printf().
 /
 /  0: Disable string functions.
@@ -38,7 +38,7 @@
 /  f_findnext(). (0:Disable, 1:Enable 2:Enable with matching altname[] too) */
 
 
-#define FF_USE_MKFS     0
+#define FF_USE_MKFS     1
 /* This option switches f_mkfs() function. (0:Disable or 1:Enable) */
 
 
@@ -55,7 +55,7 @@
 /  (0:Disable or 1:Enable) Also FF_FS_READONLY needs to be 0 to enable this option. */
 
 
-#define FF_USE_LABEL    0
+#define FF_USE_LABEL    1
 /* This option switches volume label functions, f_getlabel() and f_setlabel().
 /  (0:Disable or 1:Enable) */
 
@@ -68,7 +68,7 @@
 / Locale and Namespace Configurations
 /---------------------------------------------------------------------------*/
 
-#define FF_CODE_PAGE    932
+#define FF_CODE_PAGE    437
 /* This option specifies the OEM code page to be used on the target system.
 /  Incorrect code page setting can cause a file open failure.
 /
@@ -97,7 +97,7 @@
 */
 
 
-#define FF_USE_LFN      0
+#define FF_USE_LFN      2
 #define FF_MAX_LFN      255
 /* The FF_USE_LFN switches the support for LFN (long file name).
 /
@@ -190,8 +190,8 @@
 /  funciton will be available. */
 
 
-#define FF_MIN_SS       512
-#define FF_MAX_SS       512
+#define FF_MIN_SS       4096
+#define FF_MAX_SS       4096
 /* This set of options configures the range of sector size to be supported. (512,
 /  1024, 2048 or 4096) Always set both 512 for most systems, generic memory card and
 /  harddisk. But a larger value may be required for on-board flash memory and some
@@ -230,16 +230,16 @@
 /  buffer in the filesystem object (FATFS) is used for the file data transfer. */
 
 
-#define FF_FS_EXFAT     0
+#define FF_FS_EXFAT     1
 /* This option switches support for exFAT filesystem. (0:Disable or 1:Enable)
 /  To enable exFAT, also LFN needs to be enabled. (FF_USE_LFN >= 1)
 /  Note that enabling exFAT discards ANSI C (C89) compatibility. */
 
 
-#define FF_FS_NORTC     0
+#define FF_FS_NORTC     1
 #define FF_NORTC_MON    1
 #define FF_NORTC_MDAY   1
-#define FF_NORTC_YEAR   2018
+#define FF_NORTC_YEAR   2000
 /* The option FF_FS_NORTC switches timestamp functiton. If the system does not have
 /  any RTC function or valid timestamp is not needed, set FF_FS_NORTC = 1 to disable
 /  the timestamp function. Every object modified by FatFs will have a fixed timestamp
